src/data/faq.json

```
@@ -101,6 +101,154 @@
+                    },
+                    {
+                        "title": "How is the proof of vaccination verified?",
+                        "anchor": "vac_cert_verification",
+                        "active": true,
+                        "textblock": [
+                            "The vaccination certificate (QR code) is used digitally via the CovPass-App or the Corona-Warn-App (CWA) or alternatively as a machine-readable print. The vaccination certificate contains only information about the vaccination status, the name of the vaccinated person, and the date of birth. For service providers who want to check the vaccination status, there will be a verification app to check the vaccination certificate. This allows the vaccination status to be scanned in a similar way to a barcode of a flight or train ticket. Alternatively, proof with the paper-based vaccination document is possible."
+                        ]
+                    },
+                    {
+                        "title": "Why has no joint EU-wide project been called out for proposal for a certificate?",
+                        "anchor": "vac_cert_eu",
+                        "active": true,
+                        "textblock": [
+                            "A joint EU call for proposals would have taken too much time and would have been difficult to implement due to the different vaccination information systems in the Member States. The EU approach is about regulating a recognition framework. Any cross-border pilots could be accompanied by the EU Commission. When implementing the Digital Proof of Vaccination in Germany, the EU requirements are taken into account from the outset."
+                        ]
+                    },
+                    {
+                        "title": "Does vaccination status affect the calculation of the risk? For example, if a vaccinated person has become infected and then warns other users?",
+                        "anchor": "vac_cert_risk_assesment",
+                        "active": true,
+                        "textblock": [
+                            "No, the risk calculation in the app does not take into account whether you are vaccinated against or recovered from a corona infection. The functions are independent of each other. The quality of your risk assessment does not change depending on whether you choose to save your vaccination certificate in the application, which underlines that this is completely voluntary. However, your personal vaccination status may affect your interpretation of the risk warning."

```

Do we want to mention the case that you recovered from corona here? It was not mentioned before  and is also not included in the title of the entry. 

```suggestion
                            "No, the risk calculation in the app does not take into account whether you are vaccinated against corona. The functions are independent of each other. The quality of your risk assessment does not change depending on whether you choose to save your vaccination certificate in the application, which underlines that this is completely voluntary. However, your personal vaccination status may affect your interpretation of the risk warning."
```

---

Hm, this is theoretically something that the app could possibly store based on past positive PCR tests… but it might be better to leave it out here, agreed.

---

We can extend it ones the app does this (I think that's planned for 2.4).